components/brave_rewards/browser/rewards_service.h

```diff
@@ -227,17 +227,13 @@ class RewardsService : public KeyedService {
       GetAutoContributionAmountCallback callback) = 0;
   virtual void GetPublisherBanner(const std::string& publisher_id,
                                   GetPublisherBannerCallback callback) = 0;
+
+  // DEPRECATED: Use `SendContribution` instead.
   virtual void OnTip(const std::string& publisher_key,
```

Comment:
Now that we don't need it, can we remove `OnTip` (and any related unused paths in the ledger) rather than depracte it? Totally fine if in a separate PR.